src/utils/leak.js

```diff
@@ -160,7 +160,7 @@ function materializeObject(realm: Realm, object: ObjectValue, getCachingHeapInsp
                 descriptor.configurable !== targetDescriptor.configurable
               ) {
                 generator.emitDefineProperty(object, name, descriptor);
-              } else {
+              } else if (!object.$IsClassPrototype) {
```

Comment:
I am a bit surprised about this check here deeply nested on `object`. I don't understand why only normal properties of class properties should not be emitted?